parley/src/main/java/nu/parley/android/Parley.java

```diff
@@ -242,6 +244,14 @@ public static void disableOfflineMessaging() {
         getInstance().messagesManager.disableCaching();
     }
 
+    public static void disableRequestPushPermission() {
+        getInstance().doRequestPushPermission = false;
+    }
+
+    public static Boolean doRequestPushPermission() {
+        return getInstance().doRequestPushPermission;
+    }
+
```

Comment:
Please change this one to a single function: `setRequestNotificationPermission(boolean)`